test/performance/kshelper.py

```diff
@@ -51,48 +50,58 @@ def load_kubeconfig(self, kubeconfig, context):
         
         api_client = kubernetes.client.ApiClient()
         core_v1_api = kubernetes.client.CoreV1Api(api_client)
+        app_v1_api  = kubernetes.client.AppsV1Api(api_client)
         custom_objects_api = kubernetes.client.CustomObjectsApi(api_client)
 
         os.remove(tmp_config)
-        return core_v1_api, custom_objects_api
+        return core_v1_api, app_v1_api, custom_objects_api
 
-    def get_crd_time(self, grp, v, pl, name=None, check=False):
+    def get_custom_objects_time(self, fname, grp, v, pl, name=None, ns=None, bindingPolicy=None, check=False, single=True):
         try:
-            if name != None and check == False:
-                api_response = self.custom_objects_api.list_cluster_custom_object(group=grp, version=v, plural=pl, field_selector='metadata.name=' + name)
+            if name == None and ns != None and bindingPolicy != None:
+               api_response = self.custom_objects_api.list_namespaced_custom_object(group=grp, version=v, plural=pl, namespace=ns, label_selector='transport.kubestellar.io/originOwnerReferenceBindingKey=' + bindingPolicy)
+            elif  name != None and ns != None and bindingPolicy != None:
+               api_response = self.custom_objects_api.list_namespaced_custom_object(group=grp, version=v, plural=pl, namespace=ns, label_selector='transport.kubestellar.io/originOwnerReferenceBindingKey=' + bindingPolicy)
+            elif name != None and ns != None and bindingPolicy == None:
+               api_response = self.custom_objects_api.list_namespaced_custom_object(group=grp, version=v, plural=pl, namespace=ns, field_selector='metadata.name=' + name)
```

Comment:
Why use `list` instead of `get` here, since the name is known?

Reply:
Fixed! using `get` instead to improve performance